docs/customizing.rst

```diff
@@ -6,75 +6,101 @@ Customizing
 ===========
 
 Flask-SQLAlchemy defines sensible defaults. However, sometimes customization is
-needed. Two major pieces to customize are the Model base class and the default
-Query class.
+needed. There are various ways to customize how the models are defined and
+interacted with.
 
-Both of these customizations are applied at the creation of the :class:`SQLAlchemy`
+These customizations are applied at the creation of the :class:`SQLAlchemy`
 object and extend to all models derived from its ``Model`` class.
 
+
 Model Class
 -----------
 
-Flask-SQLAlchemy allows defining a custom declarative base, just like SQLAlchemy,
-that all model classes should extend from. For example, if all models should have
-a custom ``__repr__`` method::
+SQLAlchemy models all inherit from a declarative base class. This is exposed
+as ``db.Model`` in Flask-SQLAlchemy, which all models extend. This can be
+customized by subclassing the default and passing the custom class to
+``model_class``.
 
-    from flask_sqlalchemy import Model  # this is the default declarative base
-    from flask_sqlalchemy import SQLAlchemy
+For example, to allow testing model instances for equality based on their
+primary keys::
+
+    from flask_sqlalchemy import Model, SQLAlchemy
+    import sqlalchemy as sa
 
-    class ReprBase(Model):
-         def __repr__(self):
-             return "<{0} id: {1}>".format(self.__class__.__name__, self.id)
+    class EqModel(Model):
+        def __eq__(self, other):
```

Comment:
is `__eq__` a good example? pretty sure sqlalchemy already has a decent `__eq__` implementation

Reply:
Good point. SQLAlchemy doesn't do anything specific for `__eq__` but the model cache results in the same instances anyway. My code I pulled this from was actually for something more complicated, need to figure out a different example.

Reply:
I'll just drop the example, can't think of a good one besides the column example below.